fury/tests/test_material.py

```diff
@@ -1,12 +1,14 @@
 from fury import actor, material, window
+from fury.io import load_image
 from fury.optpkg import optional_package
-import fury.testing as ft
+from nibabel.tmpdirs import TemporaryDirectory
```

Comment:
Please, do not use nibabel. you have the same feature with standard python

Reply:
Changed in commit [d05089b](https://github.com/fury-gl/fury/pull/488/commits/d05089b63cae5af584e751925d4707b18086bcf9)